Front-End/challenge2/script.js

```diff
@@ -1,7 +1,78 @@
+/* eslint-disable linebreak-style */
 /*
 TO-DO:
 
 - Modify this file only
 - The calculator should be completely functional
 
 */
+const display = document.getElementById('display');
+
+function calculate(n1, operator, n2) {
+	let result = '';
+	const num1 = parseInt(n1, 10);
+	const num2 = parseInt(n2, 10);
+	if (operator === 'add') {
```

Comment:
This might be a perfect use-case for JavaScript's `switch` statement, since you're only comparing equality.